packages/clerk-js/src/ui/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -37,9 +42,48 @@ export const PhoneInput = (props: PhoneInputProps) => {
   const selectedCountryOption = React.useMemo(() => {
     return countryOptions.find(o => o.country.iso === selectedIso) || countryOptions[0];
   }, [selectedIso]);
+  const dynamicPadding = selectedCountryOption.country.code.length * 5; // this to calculate the padding of the input field depending the length of country code
```

Comment:
@nikosdouvlis this is not for the country code dropdown but for the input value. We want to avoid this. Currently it has a fixed padding which is not good for such cases

<img width="453" alt="Screen Shot 2022-10-31 at 1 47 15 PM" src="https://user-images.githubusercontent.com/23478420/199001006-429a84ee-13ef-42d2-960f-898f8710f538.png">

Reply:
Let's try and fix this before merging. It's more prominent when adding a phone number in `UserProfile`